tests/settings.py

```diff
@@ -42,6 +48,9 @@ def _load_yaml(path):
     return Settings(
         path=path,
         nodes=nodes,
+        options=Options(
+            enable_intermediate_model_removal=data['enable_intermediate_model_removal']
```

Comment:
I think we could also set it by default to False for skaffold file (`line 75`) and maybe add a comment like "option blabla is not supported when parsing skaffold file".

Or maybe directly create a separate PR to get rid of the network configuration defined from skaffold file? This is not very useful and I don't think it's worth it to maintain.

Reply:
Does this work? https://github.com/SubstraFoundation/substra-tests/pull/44/commits/f82d199a04bc4c85728f2d8f3a37a99aa8788283

Reply:
Not sure as there is no default value.
I will create a separate PR to remove the skaffold mode, it will solve this.